### PR TITLE
port vad 

### PR DESCRIPTION
**Pull request by fayejf**
✅  Port config file 
 ✅  Port VAD training script (need to update data processing instruction)
 ✅  Port freesound downloading scripts
 ✅  Port processing VAD data script

Next PR
- [ ] Port VAD Inference (VAD inference will be included in notebook tutorial as in 0.11 to make better demonstration) 
- [ ] Port notebook tutorials
- [ ] Tests

Signed-off-by: fayejf <fayejf07@gmail.com>

### CODE REVIEW

File: examples/asr/speech_to_label_vad.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+# Preparing the dataset
+[TODO]
```
Comment (titu1994) [inline examples/asr/speech_to_label_vad.py:17]: Todo? Or is this already done below?

File: examples/asr/speech_to_label_vad.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import pytorch_lightning as pl
+
+from nemo.collections.asr.models import EncDecClassificationModel
+from nemo.core.config import hydra_runner
+from nemo.utils import logging
+from nemo.utils.exp_manager import exp_manager
+
+
+@hydra_runner(config_path="conf", config_name="matchboxnet_3x1x64_vad")
+def main(cfg):
```
Comment (titu1994) [inline examples/asr/speech_to_label_vad.py:54]: Overall, other than the header above, this file is identical to speech_to_label.py. @blisc Do you think we should merge this file with speech_to_text.py and just add the header portion to switch the config?

Comment (fayejf) [reply examples/asr/speech_to_label_vad.py:54]: Yes, we might have different infer/eval script for different prediction methods and metrics. But for the training process, it's identical except the header.

File: examples/asr/speech_to_label_vad.py (hunk 1 above)
Comment (fayejf) [reply examples/asr/speech_to_label_vad.py:17]: Oops. Sorry, I forgot to update this part. Will fix it.

File: examples/asr/speech_to_label_vad.py (hunk 2 above)
Comment (titu1994) [reply examples/asr/speech_to_label_vad.py:54]: Ok let's merge the train script with above. Just append the docstring for vad and add two sections 1) Speech Commands 2) Voice activity detection with the respective commands underneath them.

Comment (fayejf) [reply examples/asr/speech_to_label_vad.py:54]: Good idea. `(config_path="conf", config_name="matchboxnet_3x1x64_vad")`  Do we want to keep it or make the config file mandatory? I think the second one is general?

Comment (titu1994) [reply examples/asr/speech_to_label_vad.py:54]: Keep the default as what it is inside speech to text.py. Add a line to the docs about how to switch that for vad config

**Comment by titu1994** [comment]
@fayejf Overall looks fine, but ill hold of on merging until I get conformation we can merge new models at this point

**Pull request merged**